package.json

```diff
@@ -1,5 +1,8 @@
 {
   "name": "@nib-styles/header-small",
   "version": "0.0.1",
```

Comment:
I can't think of anywhere non-react. It was made at the same time as the react component :)